Sources/NIOSSL/SSLContext.swift

```diff
@@ -411,7 +424,7 @@ extension NIOSSLContext {
             throw BoringSSLError.unknownError(errorStack)
         }
         #elseif os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
-        CNIOBoringSSL_SSL_CTX_set_custom_verify(context, SSL_VERIFY_PEER, securityFrameworkCustomVerify)
+        // We do this later, in createConnection.
```

Comment:
this looks slightly odd to `#elseif` a certain set of platforms here just to put a comment in there. Shouldn't this just be `#else` or no `#else` at all and a comment "for non-Linux platforms, we do this later, in createConnection."?